apps/mull-api/src/database/seeders/database.seeder.ts

```diff
@@ -0,0 +1,28 @@
+import { Connection } from 'typeorm';
+import { Factory, Seeder } from 'typeorm-seeding';
```

Comment:
```suggestion
import { Factory, Seeder } from 'typeorm-seeding';
import { Event } from '../../app/entities';
```

running the seeder was broken without with line

Reply:
fixed